pkg/hoist/hoist_launchable_test.go

```diff
@@ -428,7 +431,7 @@ func TestDisableWithFailingDisable(t *testing.T) {
 func TestDisableWithPassingDisable(t *testing.T) {
 	// This test's behavior is not dependent on whether the pod is a legacy or uuid pod
 	hl, _ := FakeHoistLaunchableForDirLegacyPod("successful_scripts_test_hoist_launchable")
-	err := hl.Disable()
+	err := hl.Disable(terminationGracePeriod)
```

Comment:
I don't see any tests that are exercising the new termination grace period (and I suggest you use a lower value than 1 hour when you test it 😉 ).